Looks like "git fetch refs" broken by checkout@v2

fatal: Refusing to fetch into current branch refs/heads/release/1.0 of non-bare repository
15
##[error]Docker run failed with exit code 128

(git is no longer in detached state, see https://github.com/actions/checkout/releases/tag/v2.0.0)

--- a/README.md
+++ b/README.md
@@ -26,7 +26,7 @@ jobs:
   pull-request:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v1
     - name: pull-request
       uses: repo-sync/pull-request@v2
       with:
@@ -47,7 +47,7 @@ jobs:
   pull-request:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v1
     - name: pull-request
       uses: repo-sync/pull-request@v2
       with:

